packages/blockchain-wallet-v4-frontend/src/data/components/dex/sagas.ts

```diff
@@ -346,7 +392,18 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
       }
     }
 
-    if (!counterToken || !baseToken || !getValidSwapAmount(baseTokenAmount)) return
+    if (
+      !counterToken ||
+      !baseToken ||
+      (!getValidSwapAmount(baseTokenAmount) && !getValidSwapAmount(counterTokenAmount))
```

Comment:
Trivial but this is the same as 
`!(getValidSwapAmount(baseTokenAmount) || getValidSwapAmount(counterTokenAmount)`

Reply:
agreed